src/themes/light.scss

```diff
@@ -308,6 +308,14 @@
 
   --wxc-date-input--calendar-picker-indicator--filter: invert(60%);
 
+  --wxc-adaptive-cards--action--hover--border: none;
```

Comment:
this will force the border of that element to "none", even if that element has a different border.

I would suggest changing this to "inherit". Same for foreground.

But are these 2 variabiles used anywhere?

Reply:
I now put them in Component.scss